src/test/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ScanCommandTest.java

```diff
@@ -172,6 +172,20 @@ public void testIndividualFileMatchingValidValue() throws IntegrationException {
         List<String> commandList = createCommandList();
         assertIndividualFileMatching(commandList, IndividualFileMatching.BINARY);
     }
+    
+    @Test
+    public void testIsRapidSignatureScan() throws IntegrationException {
+    	scanBatchBuilder.rapid(true);
+    	List<String> commandList = createCommandList();
+    	assertEquals(true, commandList.contains("--no-persistence"));
+    }
+    
+    @Test
+    public void testIsNotRapidSignatureScan() throws IntegrationException {
+    	scanBatchBuilder.rapid(false);
+    	List<String> commandList = createCommandList();
+    	assertEquals(false, commandList.contains("--no-persistence"));
+    }
```

Comment:
Just curious why assertEquals VS assertTrue or assertFalse? Result is the same, just wondering.